tests/components/group/test_sensor.py

```diff
@@ -243,15 +243,15 @@ async def test_reload(hass: HomeAssistant) -> None:
     assert hass.states.get("sensor.second_test")
 
 
-async def test_sensor_incorrect_state(
+async def test_sensor_ignore_non_numeric(
     hass: HomeAssistant, caplog: pytest.LogCaptureFixture
 ) -> None:
-    """Test the min sensor."""
+    """Test that non numeric values are ignored in a group."""
     config = {
         SENSOR_DOMAIN: {
             "platform": GROUP_DOMAIN,
             "name": "test_failure",
-            "type": "min",
+            "type": "max",
```

Comment:
Not that it matters but not sure why you're changing this?

Reply:
I changed the type to "max" because it makes the asserts at line 273 and line 281 expect different values which I felt slightly improved the test case.